Core/Config.h

```diff
@@ -31,6 +31,12 @@ extern const char *PPSSPP_GIT_VERSION;
 
 extern bool jitForcedOff;
 
+// should this be here?!
```

Comment:
Would be better to put them in ConfigValues.h.  I'd recommend an `enum class` although you'd have to do casts, it'd namespace the values better.

-[Unknown]

Reply:
I cleaned this up in 42873654117579a057b8741b203c17a71d09a9ef .